test/hex_web/rate_limit_test.exs

```diff
@@ -0,0 +1,63 @@
+defmodule HexWeb.RateLimitTest do
+  use ExUnit.Case, async: false
+  use Plug.Test
+
+  defmodule Hello do
+    # need to use Phoenix.Controller because RateLimit.Plug uses ErrorView
+    # which in turn requres `plug :accepts`
+    use Phoenix.Controller
+
+    plug :accepts, ~w(json)
```

Comment:
`RateLimit.Plug` works just for json format (which I guess is OK for now, as it's only being used in the :api pipeline). This line seems to be the culprit: https://github.com/hexpm/hex_web/blob/626ac5dddf70b99ee326e49d11dd8a35924a9917/web/views/error_view.ex#L7 - basically for html views the `message` key in assigns is ignored and instead a message based on status code is used. And since that is not defined for HTTP 429 (https://github.com/hexpm/hex_web/blob/626ac5dddf70b99ee326e49d11dd8a35924a9917/web/views/error_view.ex#L32) it's just blank line. Again, probably not a huge deal right now but I might tackle it later in a separate PR